pkgs/applications/misc/openambit/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv
+, cmake
+, fetchFromGitHub
+, libusb1
+, makeWrapper
+, python3
+, qtbase
+, qttools
+, udev
+, wrapQtAppsHook
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "openambit";
+  version = "0.5";
+
+  src = fetchFromGitHub {
+    owner = "openambitproject";
+    repo = pname;
+    rev = version;
+    sha256 = "1074kvkamwnlkwdajsw1799wddcfkjh2ay6l842r0s4cvrxrai85";
+  };
+
+  nativeBuildInputs = [ cmake makeWrapper qttools wrapQtAppsHook ];
+  buildInputs = [ libusb1 python3 qtbase udev zlib ];
+
+  cmakeFlags = [ "-DCMAKE_INSTALL_UDEVRULESDIR=${placeholder "out"}/lib/udev/rules.d" ];
+
+  postInstall = ''
+    install -m755 -D $src/tools/openambit2gpx.py $out/bin/openambit2gpx
+
+    mv -v $out/lib/udev/rules.d/libambit.rules \
+          $out/lib/udev/rules.d/20-libambit.rules
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Helps fetch data from Suunto Ambit GPS watches";
+    homepage = "https://github.com/openambitproject/openambit/";
+    license = licenses.gpl3Plus;
```

Comment:
At a quick glance through the repo, I don't actually see any license claimed. In that case, it would be unfree.

Reply:
There are a few COPYING files in the sub-projects that states GPL-3. Also the file headers states GPL-3.